### PR TITLE
Updates re MD5/SHA1 usage.

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
## Summary

Add recommendation  to avoid MD5/SHA1.

Fixes 1572540

Contributes to dotnet/docs#13441

@Thraka  @lizcasey 

Andy - Because of the number of files involved, I'm going to do more than one PR. This one and the next one will be about 25 files.

### CODE REVIEW

**Comment by Thraka** [comment]
Looks good, thanks!

**Comment by Thraka** [comment]
Will just leave some time for @karelz to do a quick review he has time.

**Comment by mairaw** [comment]
@bartonjs this is related to the security project that we discussed more than a year ago. Would you be the right person to review this?

Also, the changes where the sentence was added as the only summary for some APIs is wrong. The summary should explain what the API is and then the remarks for that API or the remarks for the enumeration in the case of enum fields should call out that.

**Comment by Thraka** [comment]
> @bartonjs this is related to the security project that we discussed more than a year ago. Would you be the right person to review this?
> 
> Also, the changes where the sentence was added as the only summary for some APIs is wrong. The summary should explain what the API is and then the remarks for that API or the remarks for the enumeration in the case of enum fields should call out that.

Good catch!

File: xml/System.Security.Authentication/HashAlgorithmType.xml [hunk 1]
```diff
@@ -76,7 +76,7 @@
       </ReturnValue>
       <MemberValue>32771</MemberValue>
       <Docs>
-        <summary>The Message Digest 5 (MD5) hashing algorithm.</summary>
+        <summary>The Message Digest 5 (MD5) hashing algorithm. Due to collision problems with MD5, Microsoft recommends SHA256.</summary>
```
Comment (Thraka) [inline xml/System.Security.Authentication/HashAlgorithmType.xml]: These two changes in this file should also move to `remarks`.

I think after this change it should be good to merge. Thanks again!

**Pull request merged**